staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -84,12 +84,20 @@ func MergeAPIResourceConfigs(
 		}
 	}
 
+	// "api/legacy=false" allows users to disable legacy api versions.
```

Comment:
can we avoid to leak this old legacy code into k8s.io/apiserver?

Reply:
Done